docs/detections/rules-ui-manage.asciidoc

```diff
@@ -35,6 +36,18 @@ To filter the rules list, enter a search term in the search bar and press **Retu
 
 NOTE: Searches for index patterns and MITRE ATT&CK tactics and techniques must match exactly, are case sensitive, and do _not_ support wildcards. For example, to find rules using the `filebeat-*` index pattern, the search term `filebeat-*` is valid, but `filebeat` and `file*` are not because they don't exactly match the index pattern. Likewise, the MITRE ATT&CK tactic `Defense Evasion` is valid, but `Defense`, `defense evasion`, and `Defense*` are not.
 
+[float]
+[[rule-status]]
+=== Check the current status of rules
+
+The *Last response* column displays the current status of each rule, based on the most recent attempt to run the rule:
+
+* *Succeeded*: The rule was able to complete its defined search. This doesn't necessarily mean it generated an alert, just that it ran without error.
+* *Failed*: The rule encountered an error that prevented it from running. For example, a {ml} rule whose corresponding {ml} job wasn't running.
+* *Warning*: Nothing prevented the rule from running, but it still might not have returned the expected results. For example, a custom query rule tried to search an index pattern that couldn't be found in {es}.
```

Comment:
```suggestion
* *Warning*: Nothing prevented the rule from running, but it might have returned unexpected results. For example, a custom query rule tried to search an index pattern that couldn't be found in {es}.
```